lib/browser/browserStorage.ts

```diff
@@ -220,12 +221,11 @@ var storageUtil: BrowserStorageUtil = {
 
   storage: {
     set: function(name: string, value: string, expiresAt: string, options: CookieOptions): string {
-      const secure = options.secure;
-      const sameSite = options.sameSite;
+      const { sameSite, secure } = options;
       if (typeof secure === 'undefined' || typeof sameSite === 'undefined') {
         throw new AuthSdkError('storage.set: "secure" and "sameSite" options must be provided');
       }
-      var cookieOptions: CookieOptions = {
+      var cookieOptions: SetCookieOptions = {
```

Comment:
Why change the type here?

Reply:
It looks like [SetCookieOptions](https://github.com/okta/okta-auth-js/blob/master/lib/types/Cookies.ts#L25) type is intended to be used when [creating](https://github.com/okta/okta-auth-js/blob/master/lib/browser/browserStorage.ts#L243) a cooke.

Reply:
looks like just different names, we probably can remove one in the future. Now let's just keep what it is to avoid breaking changes.